Gemfile.lock

```diff
@@ -1,101 +1,132 @@
 PATH
```

Comment:
Could you gitignore this file? Gemfile.lock is tracked for applications, but gems don't distribute a lock file. We could control that at publishing time, but most gems seem to instead go with the gitignore approach.